include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -65,7 +65,22 @@ class MGConstrainedDoFs : public Subscriptor
   template <int dim, int spacedim>
   void initialize(const DoFHandler<dim,spacedim> &dof,
                   const typename FunctionMap<dim>::type &function_map,
-                  const ComponentMask &component_mask = ComponentMask());
+                  const ComponentMask &component_mask = ComponentMask()) DEAL_II_DEPRECATED;
+
+  /**
+   * Fill the internal data structures with information
+   * about Dirichlet boundary dofs.
+   *
+   * The initialize() function has to be called additionally
+   * to set hanging node constraints.
```

Comment:
additionally -> in addition to

what does "set hanging node constraints" refer to? Is there a function `set_hanging_node_constraints()`? If so, maybe use that name.

I find the sentence a bit too vague because it only says that one has to do other things as well, but does not specify a temporal order. How about something of the form
```
  Before calling this function, you also have to call `initialize()` and `set_hanging_node_constraints()`.
```